maven_test.go

```diff
@@ -435,3 +437,60 @@ func prepareMavenSetupTest(t *testing.T, homeDir string) func() {
 		restoreDir()
 	}
 }
+
+func TestMavenConfig(t *testing.T) {
```

Comment:
I suggest creating a common function between these two test since they share a lot of code